### PR TITLE
Added Ayu Theme.

### PR DESCRIPTION
**Pull request by botex98**

### CODE REVIEW

File: ayu.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Ayu Themes
+
+![AyuMirage](images/AyuMirage.png)
+![AyuDark](images/AyuDark.png)
+![AyuLight](images/AyuLight.png)
+
+## Credit
```
Comment (rjcarneiro) [inline ayu.md:7]: This header doesn't belong here.

File: ayu.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+[Theme: Ayu Theme](https://github.com/ayu-theme)
+
+[Scheme creator: joshtynjala](https://gist.github.com/joshtynjala/10dbdd4e449027fe9723b1d9f553c0bd)
+
+## Settings
+
+```json
+{
+    "colorScheme": "AyuMirage"
```
Comment (rjcarneiro) [inline ayu.md:17]: The `colorScheme` doesn't match the ones defined, should be `Ayu Mirage` instead.

File: ayu.md (hunk 1 above)
Comment (botex98) [reply ayu.md]: done.

File: ayu.md (hunk 2 above)
Comment (botex98) [reply ayu.md]: done.

File: ayu.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+```json
+{
+    "colorScheme": "Ayu Mirage"
+}
+```
+
+## Fonts
+
+- Cascadia Mono
```
Comment (rjcarneiro) [inline ayu.md:17]: Hi @botex98, by any changes, don't have the font? There's a `fonts/` folder where we stored all the fonts... this way, anyone can follow the settings and the set up is much more easy.

File: ayu.md (hunk 2 above)
Comment (rjcarneiro) [reply ayu.md]: Thank you!

File: ayu.md (hunk 1 above)
Comment (rjcarneiro) [reply ayu.md]: Thank you!

File: ayu.md (hunk 3 above)
Comment (botex98) [reply ayu.md:17]: It's really easy to find and they're continously working on it...
Should I add the followin link to the font in the `ayu.md` file?
https://github.com/microsoft/cascadia-code

Comment (botex98) [reply ayu.md:17]: @rjcarneiro I've added the latest release.

**Comment by rjcarneiro** [comment]
Thanks @botex98 for your contribution 💪

**Pull request merged**